paper.tex

```diff
@@ -41,15 +41,15 @@
 does not constitute an ARG is partly attributable to the historical
 development of the term.
 Originally rigorously defined as
-a representation of the coalescent with recombination
-stochastic process as a graph, an ARG is now understood to refer to any
+a stochastic process (the coalescent with recombination) together with its 
+representation, an ARG is now understood to refer to any
 concrete realisation of a recombinant genetic ancestry.
 We show that the standard description of an ARG
 in terms of common ancestor and recombination events
 (inherited from the original stochastic process definition) is
 ambiguous, fundamentally limited and an unsuitable basis for data interchange.
 We provide a simple alternative definition of an ARG in which a node
-corresponds to one of an individuals monoploid genomes, an edge
```

Comment:
I think we need to be ultra clear and specific about this though, even if it is technically redundant. People will skip over it without thinking it through properly if we just say "a genome"

Reply:
That's a fair point. I wanted to make sure that it wasn't just diploids that this applied to, but your previous phrasing didn't state that, so I guess that's OK. It just needs the apostrophe in there then.